### ISSUE TITLE
Problem in Arabic language diacritics 

### ISSUE THREAD

**Issue by ammobini** [nvaccess/nvda#12878]
I am writing to report an issue that exists in NVDA 2021 but that didn't exist in older NVDAs like 2017.
NVDA 2021 reads the Arabic diacritics -which can be considered as the vowels sounds- , but in some applications it doesn't read them out
for example, suppose an Arabic word with diacritics in Microsoft word, the diacritics can be read out when I Move the review cursor to the next character by right/left arrow to move between characters.
If I navigate the same word in IE or Mozilla Firefox or etc, and move with the arrow keys, NVDA ignores these diacritics and just reads out the letters.
a word like فَاطِمَةً is an example.
Try to copy the word above in a notepad text file and move between the letters. what you will notice: NVDA will just move between the letters and ignores the diacritics but if you try in Microsoft Word diacritics also will be sound.
If you need more information don't hesitate to tell me.

**Comment by Adriani90** [comment]
This is not reproducibble with NVDA 2025.3 in Firefox, Notepad or MS Word. All letters are read correctly. In Notepad, letters are not joined, but in MS Word and / or edit fields in Firefox, some letters with diacritics are combined. But in any case NVDA does not ignore any diacritics.

I am closing as works for me. If you are having this issue with NVDA 2025.3 still, please comment and we can reopen.
